code/modules/mob/living/carbon/alien/humanoid/queen.dm

```diff
@@ -45,6 +46,16 @@
 	smallsprite.Grant(src)
 	return ..()
 
+/mob/living/carbon/alien/humanoid/royal/queen/get_status_tab_items()
+	. = ..()
+	. += ""
+	. += "Blocked Shuttle Timer: [round(timeleft(time_to_shuttle) / 600, 1)] minutes" //weird conversion but works
```

Comment:
this may runtime when the timer is deleted

Reply:
it doesn't runtime but it is true it shouldn't show up